Changelog

```diff
@@ -0,0 +1,32 @@
+1.0.0 (October 3rd, 2025)
```

Comment:
```suggestion
1.0.0 (October 6th, 2025)
```

(In fact, if this PR is not merged today this should be change to the day when this PR gets merged)